.circleci/config.yml

```diff
@@ -28,6 +28,35 @@ apt-run:  &apt-install
       libgeos-dev \
       otf-freefont
 
+conda36-run:  &conda36-install
+  name: Install conda
+  command: |
+    wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh \
+      -O miniconda.sh
+    chmod +x miniconda.sh && ./miniconda.sh -b -p $HOME/miniconda
+    export PATH="$HOME/miniconda/bin:$PATH"
+    echo $PATH
+    conda update --yes --quiet conda
+
+    # Configure the conda environment and put it in the path using the
+    # provided versions
+    conda create -n testenv --yes --quiet python
+    source activate testenv
+
+conda27-run:  &conda27-install
```

Comment:
All of these are templated so you don't need to write out two copies for Python 3 and 2. There should not be a separate 36 and 27 template. You can use an environment variable to differentiate if necessary (see the `NUMPY` variable for example).